platform/ui/src/components/downloadDialog/DownloadDialog.styl

```diff
@@ -0,0 +1,102 @@
+@import './../../design/styles/common/global.styl'
+@import './../../design/styles/common/form.styl'
+@import './../../design/styles/common/button.styl'
+
+.DownloadDialog
+  color: var(--text-secondary-color);
+  filter: drop-shadow(0 0 3px var(--ui-gray-darkest));
+  border: none;
+  border-radius: 8px;
+  width: inherit;
+  padding: 15px;
+  background: transparent;
+  z-index: 1080 !important;
+
+  .title
+    margin: 0;
+    font-weight: bold;
+
+  .file-info-container
+    display: flex;
+    flex-direction: row;
+    margin: 20px 0;
+    border-radius: 5px;
+    padding: 20px 10px 0;
+    background-color: #16202b;
+
+    .form-control.input-ohif
+      padding: 6px 12px;
+
+    @media screen and (max-width: 1023px)
+      flex-direction: column;
+
+    .col
+      flex-grow: 1;
+
+      .input-ohif
+        margin-left: 15px;
+        @media screen and (max-width: 1023px)
+          margin-left: 0;
+          margin-top: 5px;
+          width: 100% !important;
+
+      .file-type
+        .select-ohif
+          margin-left: 17px;
+          @media screen and (max-width: 1023px)
+            margin-left: 0;
+            width: 100%!important;
```

Comment:
Missing space. Also, not sure of `!important` usage throughout file.

Reply:
> See comments for code specific suggestions.
> 
> Here is some additional feedback after playing with the deploy preview:
> 
> * If I erase the name for the image, it downloads as `.png` or `.jpeg`. Can we use a default name if the value is an empty string?
> * For wider screens, the image preview is not centered; can we center it?
> * For smaller screens, the image preview can leak out of it's box.
>   
>   * Maybe a `max-width: 100%` or something? Could use
> 
> @igoroctaviano, everything is looking _really good_. I think we're very close to having this merged. Thank you for your hard work 🙏

So, I have added a default filename like you said, good catch! I also change the way i was aligning the image preview, now it seems to be centered. The only one I could not replicate was the small screen leak, can you share some steps? maybe im missing one.

Reply:
> Missing space. Also, not sure of `!important` usage throughout file.

done! thx.